apps/desktop-wallet/src/storage/addresses/addressesAdapters.ts

```diff
@@ -21,11 +21,6 @@ import { createEntityAdapter } from '@reduxjs/toolkit'
 import { Address } from '@/types/addresses'
 
 export const addressesAdapter = createEntityAdapter<Address>({
-  selectId: (address) => address.hash,
-  sortComparer: (a, b) => {
-    // Always keep main address to the top of the list
-    if (a.isDefault) return -1
-    if (b.isDefault) return 1
-    return (b.lastUsed ?? 0) - (a.lastUsed ?? 0)
-  }
+  selectId: (address) => address.hash
+  // TODO: Find a new way to get sorted addresses
```

Comment:
I would like to work on this in a follow up, this PR already grew more than I wanted.